pyproject.toml

```diff
@@ -120,7 +120,7 @@ netaddr = "~1.3.0"
 # Library of network related utilities including network driver mappings and Jinja Filters
 # Note: netutils is limited in scope, dependencies, and observes semver, as such
 #       we permit a looser (^) version constraint here.
-netutils = "^1.6.0"
+netutils = "^1.14.0"
```

Comment:
Any relevant bug fixes in 1.14.1 such that we'd want to set it as the minimum instead?

Reply:
No, there was a change into using "show run" vs "show running-config" that should be benign